src/main/java/com/marklogic/dataexplorer/command/DeployCommand.java

```diff
@@ -25,4 +30,18 @@ protected void copyParametersToSystemProperties() {
 			System.setProperty("mlRestPort", port.toString());
 		}
 	}
+
+	protected void configureDeployerToNotDeployMl9SpecificPrivileges(ClasspathDeployer deployer) {
```

Comment:
This does the real work, everything else was just moving furniture around to allow this to work.